e2e/button/button.spec.cy.tsx

```diff
@@ -0,0 +1,14 @@
+import { mount } from '@cypress/vue'
```

Comment:
Why move to top level `/e2e` directory?

Reply:
If we keep that inside `packages/components`, we will need to update `jest.config` for skipping cypress tests in this case I think we should keep it under `e2e`

Reply:
I can help you, could you move it to `packages/components` and I will update `tsconfig.json`

Reply:
It was intended to keep them outside the package. Please just accept it as-is.